clojure-mode.el

```diff
@@ -524,7 +524,9 @@ replacement for `cljr-expand-let`."
 \\{clojure-mode-map}"
   (clojure-mode-variables)
   (clojure-font-lock-setup)
-  (add-hook 'paredit-mode-hook #'clojure-paredit-setup))
+  (add-hook 'paredit-mode-hook #'clojure-paredit-setup)
+  (add-hook 'electric-indent-functions
```

Comment:
I'd really add some explanations here, as people are going to be confused by this bit of code for sure. 

You might also consider adding some simple unit test verifying the correct behavior when electric indent is enabled.